.github/workflows/release-please.yml

```diff
@@ -10,6 +10,7 @@ jobs:
       - uses: google-github-actions/release-please-action@v3
         with:
           release-type: go
+          token: ${{ FINCH_BOT_TOKEN }}
```

Comment:
Need PAT since `tag` and `release` created with `release-please` will not trigger subsequent workflows with `GITHUB_TOKEN`. See https://github.com/google-github-actions/release-please-action#github-credentials